modules/nf-core/diamond/blastp/meta.yml

```
@@ -59,82 +62,87 @@ output:
           description: |
             Groovy Map containing sample information
             e.g. [ id:'test', single_end:false ]
-          pattern: "*.{blast,blast.gz}"
       - "*.{blast,blast.gz}":
-          type: map
+          type: file
           description: File containing blastp hits
           pattern: "*.{blast,blast.gz}"
+          ontologies:
+            - edam: http://edamontology.org/format_3836 # BLAST XML v2 results format
   - xml:
       - meta:
           type: map
           description: |
             Groovy Map containing sample information
             e.g. [ id:'test', single_end:false ]
-          pattern: "*.{xml,xml.gz}"
       - "*.{xml,xml.gz}":
-          type: map
+          type: file
           description: File containing blastp hits
           pattern: "*.{xml,xml.gz}"
+          ontologies:
+            - edam: http://edamontology.org/format_2332 # XML
   - txt:
       - meta:
           type: map
           description: |
             Groovy Map containing sample information
             e.g. [ id:'test', single_end:false ]
-          pattern: "*.{txt,txt.gz}"
       - "*.{txt,txt.gz}":
-          type: map
+          type: file
           description: File containing hits in tabular BLAST format.
           pattern: "*.{txt,txt.gz}"
+          ontologies: []
```

I think we could use the BLAST results format here
```suggestion
          ontologies: 
              - edam: http://edamontology.org/format_1333 # BLAST results
```